docs/front-end/web-accessibility.md

```diff
@@ -0,0 +1,102 @@
+
+- Operable: Any user must be able to use controls, buttons, navigation, and other interactive elements by themselves (e.g. buttons must be clickable using mouse, keyboard, voice command, etc.). Additionally, it takes into account that disabled users will use assistive technology like voice recognition, keyboards, screen readers, and so on.
+
+- Understandable: Any user should be able to understand the content.
+
+- Robust: The content should adhere well-adopted web standards that are functional cross-browsers, now and in the future.
+
+## Guidance
+
+A lot of the obvious things can be automated, however a majority of accessibility issues that impact users are things that are difficult or impossible to programmatically determine. As developers, we are aware about the importance of making our websites/ apps accessible to all users, but the journey to achieving this can be quite complex. Here are reference resources provided in the WCAG docs which should be your first point of reference.
+
+- How to Meet [WCAG](https://www.w3.org/WAI/WCAG22/quickref/?versions=2.1) (Quick Reference)
+- Index of [ARIA Design Pattern](https://www.w3.org/WAI/ARIA/apg/) Examples
+
+Other links:
+
+- [W3C WAI Tutorials](https://www.w3.org/WAI/tutorials/)
+- [MDN Accessibility guide](https://developer.mozilla.org/en-US/docs/Learn/Accessibility/HTML])
+- [WebAIM Articles](https://webaim.org/articles/)
+- [A11yProject](https://www.a11yproject.com/)
+- [HTML5 Accessibility](https://www.html5accessibility.com/index.html)
+- [Inclusive Components](https://inclusive-components.design/)
+
+## Tools
+
+In addition to keeping up to date with constantly evolving WCAG guidelines, instances or considerations around ensuring proper color contrasts to implement keyboard navigation and screen reader compatibility can be an extensive list. This is where tools can be a win to simplify the entire process. These tools assist in automating a lot of the heavy lifting, like scanning for common accessibility issues and suggesting possible fixes. It's a good balance between saving time, addressing accessibility related issues as well educating about best practices in accessibility. Some of the tools widely used in the community:
+
+- [IBM Accessibility Checker](https://www.ibm.com/able/toolkit/tools/)
+This is the tool offered by IBM. You can find a web version (chrome and firefox extensions) and the [npm](https://www.npmjs.com/package/accessibility-checker) library on the site. The tool uses IBM's accessibility rules engine detecting WCAG 2.1 accessibility issues for web-based content or applications. Checkout [IBM Accessibility Central](https://pages.github.ibm.com/IBMa/able/) to better understand accessibility and making software accessible at IBM.
+
+IBM's Developer [guidelines and checklist](https://www.ibm.com/able/guidelines/index.html).
+
+- [Tools List by W3C](https://www.w3.org/WAI/test-evaluate/tools/list/)
+
```

Comment:
The blank lines in between list items here and above are likely to create multiple unordered lists with one list item each. Remove the blank lines to keep it to one list.